src/packages/core/variant/types.ts

```diff
@@ -3,16 +3,18 @@ import type { UmbLanguageDetailModel } from '@umbraco-cms/backoffice/language';
 import type { ScheduleRequestModel } from '@umbraco-cms/backoffice/external/backend-api';
 import type { UmbPropertyValueData } from '@umbraco-cms/backoffice/property';
 
-export interface UmbVariantPropertyValueModel extends UmbPropertyValueData {
+export interface UmbVariantDataModel {
 	culture: string | null;
 	segment: string | null;
```

Comment:
idea for a future improvement.

We have a couple of places where we set the culture and segment props. Maybe we could let them extend the UmbVariantDataModel ?